.github/workflows/compress-images.yml

```diff
@@ -0,0 +1,18 @@
+name: compress-images
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  compress:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          sudo apt install -y imagemagick optipng 
+          find -name '*.jpg' -exec mogrify -strip -compress JPEG {} +
```

Comment:
we already use https://sharp.pixelplumbing.com/ for the flow background (see https://github.com/goauthentik/authentik/blob/main/web/package.json#L25) so maybe we should use that here too?

Also I dont think these compression commands are idempotent so this CI pipeline might always fail?

Reply:
optipng is idempotent, imagemagick (either `convert` or `mogrify`) isn't